### ISSUE TITLE
FYI - bluewalker can decode messages on Linux

### ISSUE THREAD

**Issue by ikke-t** [pvvx/ATC_MiThermometer#152]
Hi,

perhaps there could be a list in Readme of links to known software which can print out the BLE broadcasts from this firmware. If someone is looking for reading this info from e.g. raspberry pi, there is bluewalker software that can read it and print it out in json or just text. I use it with telegraf to push the info to influxdb for home automation to show it from.

Hopefully this helps someone! 

https://gitlab.com/jtaimisto/bluewalker

**Comment by ikke-t** [comment]
It looks like this

```
{ 	"device": { 		"address": "a4:c1:38:fa:e3:46", 		"type": "LE Public" 	}, 	"rssi": -76, 	"time": "2021-11-26T16:40:16.74475812+02:00", "type": "mijia", 	"sensors": { 		"uuid": 6170, 		"mac": [ 			70, 			227, 			250, 			56, 			193, 			164 		], 		"temperature": 22.29, 		"humidity": 29.75, 		"voltage": 3.046, 		"level": 94, 		"counter": 62, 		"flags": 5 	} } 

 ```

**Comment by ikke-t** [comment]
Like this:

```
{
	"device": {
		"address": "a4:c1:38:fa:e3:46",
		"type": "LE Public"
	},
	"rssi": -76,
	"time": "2021-11-26T16:40:16.74475812+02:00",
      "type": "mijia",
	"sensors": {
		"uuid": 6170,
		"mac": [
			70,
			227,
			250,
			56,
			193,
			164
		],
		"temperature": 22.29,
		"humidity": 29.75,
		"voltage": 3.046,
		"level": 94,
		"counter": 62,
		"flags": 5
	}
}
```

**Comment by cimm** [comment]
I know this issue not really related to the custom firmware, but I tried reading the BLE advertisement data in Python and Go on Linux and got nowhere, a collection of random weird issues (would work once but not again for example). It's thanks to this issue I learned about bluewalker and... it works perfectly out of the box!

```
sudo btmgmt --index hci0 power off
sudo ./bluewalker -device hci0 -mijia -duration -1
```

Same use case, by the way. I am also pushing this data to InfluxDB.

Thanks for taking the time to point me (and hopefully others) in the right direction @ikke-t [ikke-t](https://github.com/ikke-t)!
